### PR TITLE
Kanban board

### PR DESCRIPTION
**Pull request by DefinitelyADev**
- Moved the Project under the Milestone in the issue template in order to be consistent with the rest of the UI
- Fixes error 500 when trying to create a User or an Organization project

### CODE REVIEW

File: models/project.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func GetProjectsConfig() []ProjectsConfig {
 // IsProjectTypeValid checks if a project typeis valid
 func IsProjectTypeValid(p ProjectType) bool {
 	switch p {
-	case ProjectTypeRepository:
```
Comment (6543) [inline models/project.go]: was removed intentional

**Review by 6543 — CHANGES_REQUESTED** [review]
instead of allowing this - we should remov UI to recat that type of projects until there is support for them

**Comment by DefinitelyADev** [comment]
@6543 Done! Also fixed the issue where the create a User or Organization project was showing.

File: models/unit.go [hunk 2]
```diff
@@ -141,10 +141,10 @@ func loadUnitConfig() {
 func (u UnitType) UnitGlobalDisabled() bool {
 	for _, ud := range DisabledRepoUnits {
 		if u == ud {
-			return true
+			return false
```
Comment (6543) [inline models/unit.go]: this make things worse ... I'll have a look at the creation menue and fix it ... sory I know you have put in time and effort

Comment (DefinitelyADev) [reply models/unit.go]: This disables the global projects from the ui. How does it make it worse?

Plus it mentions it in the [PR](https://github.com/go-gitea/gitea/pull/8346) that it fixes User and Organization projects

Comment (DefinitelyADev) [reply models/unit.go]: @6543 Sorry, I will fix the UI template. I'll try to do it today or at least tomorrow.

**Review by 6543 — APPROVED** [review]

**Pull request merged**